daemon/containerd/image_builder.go

```diff
@@ -37,7 +37,10 @@ import (
 	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
-const imageLabelClassicBuilderParent = "org.mobyproject.image.parent"
+const (
+	// Digest of the image which was the base image of the committed container.
+	imageLabelClassicBuilderParent = "org.mobyproject.image.parent"
+)
```

Comment:
`gofumpt` will probably want this to be a single `const` (not a block);

```suggestion
// Digest of the image which was the base image of the committed container.
const imageLabelClassicBuilderParent = "org.mobyproject.image.parent"
```

Reply:
Removed.